File: beer.py
#!/bin/python3
count = "bottles"
for i in list(range(99,0,-1)):
	if i == 1:
		count = "bottle"
	if i != 99:
		print(i, count, "of beer on the wall")
		print()
	print(i, count, "of beer on the wall,")
	print(i, count, "of beer")
	print()
	print("Take one down, pass it around,")
else:
	print("no more bottles of beer on the wall")
	print()
	print("Go to the store and buy some more,")
	print("99 bottles of beer on the wall")

File: beer.c
#include <stdio.h>

int i = 99;
int count() {
	printf("%i bottle", i);
	if ( i > 1 ) { printf("s"); }
}

int main() {
	for ( ;; i-- ) {
		if ( i != 0 ) {
			if ( i != 99 ) {
				count(); printf(" of beer on the wall\n\n");
			}
			count(); printf(" of beer on the wall,\n");
			count(); printf(" of beer\n\n");
			printf("Take one down, pass it around,\n");
		} else {
			printf("no more bottles of beer on the wall\n\n");
			printf("Go to the store and buy some more,\n");
			printf("99 bottles of beer on the wall\n");
			return 0;
		}
	}
}
